docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+  
+
+**Returns**
+- partialUserOp (`Promise<Partial<UserOperation>>`): A Promise resolving to `partialUserOp` which can be further sent for execution⁠.
+
+### senduserOp( )
+
+This method is used to submit a User Operation object to the User Operation pool of the client. It signs the UserOperation using an externally owned account (EOA) key and submits it to the bundler for on-chain processing. 
+
+**Usage**
+
+```ts
+const userOpResponse = await smartAccount.sendUserOp(userOp);
+
+type UserOpResponse = {
+  userOpHash: string;
+  wait(_confirmations?: number): Promise<UserOpReceipt>;
+  waitForTxHash(): Promise<UserOpStatus>;
+};
+
+const { receipt } = await userOpResponse.wait(1);
+
+type UserOpReceipt = {
+  userOpHash: string;
+  entryPoint: string;
+  sender: string;
+  nonce: number;
+  paymaster: string;
+  actualGasCost: BigNumber;
+  actualGasUsed: BigNumber;
+  success: boolean;
+  reason: string;
+  logs: Array<ethers.providers.Log>;
+  receipt: ethers.providers.TransactionReceipt;
+};
+```
+
+**Parameters**
+- userOp (`Partial<UserOperation>`, required): The `userOp` object includes essential fields like `sender`, `nonce`, `callData`, `gas` related properties.
+
+- params (`SendUserOpParams`): The `SendUserOpParams` object can contain fields such as `sessionID`, `sessionSigner`, `sessionValidationModule`, `additionalSessionData`, `batchSessionParams`, and `simulationType`. These parameters are used to customize the behavior of the `sendUserOp` method and are optional. 
+  ```ts
+  const userOpResponse = await moduleSmartAccount?.sendUserOp(userOp, {
+    sessionSigner: sessionSigner,
+    sessionValidationModule: moduleAddr,
+  });
+  ```
+  Similar to building a `userOp` we need to ensure that any modules used for additional validation or execution logic are specified in the `sendUserOp` method. Currently, this only applies for session key module requirements.
+
+:::note
+Please note that `simulationType` allows for more debugging insights about `callData` on why an internal transaction fails. It is set to "validation" by default, but can be changed to "validation_and_execution" for more detailed tracing.
+:::
+
+**Returns**
+- userOpsResponse (`UserOpResponse`): The method returns an object of type `UserOpResponse` which has a `userOpHash` and two methods: `wait()` and `waitForTxHash()`.
+The `wait()` method resolves when the user operation is dispatched by the bundler on-chain and gets mined. The `waitForTxHash()` method returns a `UserOpStatus` object which includes the transaction hash and the receipt once added on-chain.
+
+
+### sendSignedUserOp( )
+
+This method is designed to handle user operations that have already been signed. These signed operations are sent directly to the bundler for on-chain processing.
+This method is particularly useful when handling operations that have been grouped together with a multi-chain module, as it allows for the submission of these combined operations in a single request.
```

Comment:
It can also be useful when you have two different instances of smart account for example one instance of the backend just to build userOp, another instance to get it signed on front end and then dispatch signed userOp using backend instance.